src/frontend/src/optimizer/logical_optimization.rs

```diff
@@ -612,6 +612,7 @@ impl LogicalOptimizer {
         // Must push down predicates again after split over window so that OverWindow can be
         // optimized to TopN.
         plan = Self::predicate_pushdown(plan, explain_trace, &ctx);
+        plan = plan.optimize_by_rules(&LOGICAL_FILTER_EXPRESSION_SIMPLIFY);
```

Comment:
> `.map(column_self_eq_eliminate)`

If we have added `column_self_eq_eliminate` to the filter simplify method, do we need to add this rule here?

Reply:
I forget to delete it 🤣